Dockerfile

```diff
@@ -16,7 +16,15 @@ RUN apt-get update && \
   zip \
   && rm -rf /var/lib/apt/lists/*
 
+USER tfc-agent
+
+# Install tools via asdf (grabbing versions from build-harness-extensions)
+ADD --chown=tfc-agent https://raw.githubusercontent.com/mintel/build-harness-extensions/main/modules/satoshi/tf-tool-versions .
 COPY install-binaries.sh .
```

Comment:
any reason why we need this vs just bundle it into the dockerfile as an `asdf install` ?

Reply:
It's a little more complicated than that, since we also need to download asdf, run all the plugin commands from `.tool-versions` and grep out Terraform so it doesn't conflict with the agen'ts version. It could go in the Dockerfile but I feel it would be messy running it all in a single command in the Dockerfile. Reckon it's clearer this way!

Reply:
ok!